### ISSUE TITLE
Got warped away too far

### ISSUE THREAD

**Issue by NocreativityTheSecond** [Eggs-D-Studios/wos-issues#437]
### Guidelines

- [X] I checked for duplicate bug reports
- [X] I tried to find a way to reproduce the bug

### Version

Main (Production)

### What happened? What did you expect to happen?

When i warped away, for some reason i got sent into null zone, everything was glitchy, nothing worked, and i even got to noclip
![зображення_2024-09-26_152509175](https://github.com/user-attachments/assets/b3a20569-75bf-441c-b423-9edcbdcfb093)
![Знімок екрана 2024-09-26 152024](https://github.com/user-attachments/assets/39efdd49-63d9-40a2-94ce-cd96c13f0a10)

https://github.com/user-attachments/assets/de4c058d-7f53-45cd-bc59-f86008279dcc

### How can we reproduce this bug?

Basically, when you go to space, theres chance that you go to null zone

### Frequency

I can't make it happen consistently

**Comment by Hexcede** [comment]
I am not sure why I marked this as a duplicate, I didn't link the issue (oops) but it is resolved. This bug is a combination of a resolved game bug and a Roblox bug. The Roblox bug causes all of the parts to become intangible due to attached structures hundreds of thousands or millions of studs away, and the resolved game bug is the structures getting moved that far away in the first place.
